[8.x] Don't modify index mappings for Alerts indices (#6882)
elastic/security-docs: docs/reference/alert-schema.asciidoc

{elastic-sec} stores alerts that have been generated by detection rules in hidden {es} indices. In 8.x versions, the index pattern is `.alerts-security.alerts-<space-id->`. In 7.x versions, the index pattern was `.siem-signals-<space-id>`, and some field names were different. The following table includes the current names and cross-references the legacy field names.

[IMPORTANT]
====

* System indices, such as the alert indices, contain important configuration and internal data; do not change their mappings. Changes can lead to rule execution and alert indexing failures. Use <<runtime-fields, runtime fields>> instead, which allow you to add fields to existing alert and event documents.

> * Users are advised NOT to use the `_source` field in alert documents, but rather to use the `fields` option in the search API to programmatically obtain the list of fields used in these documents. Learn more about {ref}/search-fields.html[retrieving selected fields from a search].

Reviewer: Switching to 2nd person friendly tone. 
```suggestion
* We recommend to NOT use the `_source` field in alert documents, but rather the `fields` option in the search API to programmatically obtain the list of fields used in these documents. Learn more about {ref}/search-fields.html[retrieving selected fields from a search].
```
